assets/js/partials/happy_point.js

```diff
@@ -19,11 +19,16 @@ $(document).ready(function() {
     }
   }
 
+  var cookie = readCookie('greenpeace');
```

Comment:
Please add a `/* global readCookie */` on the top of this file to make Travis happy :)